sgkit/stats/aggregation.py

```diff
@@ -216,14 +241,30 @@ def allele_frequency(ds: Dataset) -> Dataset:
     return Dataset(data_vars)
 
 
-def variant_stats(ds: Dataset, merge: bool = True) -> Dataset:
+def variant_stats(
+    ds: Dataset,
+    *,
+    call_genotype_mask: str = "call_genotype_mask",
+    call_genotype: str = "call_genotype",
+    variant_allele_count: Optional[str] = None,
+    merge: bool = True,
+) -> Dataset:
     """Compute quality control variant statistics from genotype calls.
 
     Parameters
     ----------
     ds
         Genotype call dataset such as from
         `sgkit.create_genotype_call_dataset`.
+    call_genotype
+        Input variable name holding call_genotype.
+        As defined by `sgkit.variables.call_genotype`.
```

Comment:
We should probably decide on how we want to do this linking, and what kind of information goes into method doc vs variable docs. I would actually suggest we do this in a separate PR, to keep review slim. But if people feel strong to do it here, fine as well.

Reply:
FYI added links between methods and variables in the documentation.